server/src/models/paginator.py

```diff
@@ -0,0 +1,32 @@
+import math
+import typing as t
+
+
+class Paginator:
+    page = 1
+    limit = 10
+
+    def __init__(self, page: t.Optional[str], limit: t.Optional[str]):
+        try:
+            self.page = math.floor(float(page))
+        except (TypeError, ValueError):
+            pass
+        try:
+            limit = math.floor(float(limit))
+            print(limit)
+            if limit > 100:
+                self.limit = 100
+            elif limit > 0:
+                self.limit = limit
+        except (TypeError, ValueError):
+            pass
+
+    def toJSON(self, rooms: list, total_rooms: t.Union[int, float]) -> dict:
+        return {
+            'rooms': rooms,
```

Comment:
1) games*
2) paginator dont must know about what its pagniate, so remove rooms from here, 
3) if you need total count - pass it into constructor